lib/demo_web/templates/layout/live.html.leex

```diff
@@ -5,7 +5,7 @@
 <hr/>
 
 <div id="main">
-  <%= @live_view_module.render(assigns) %>
+  <%= @live_module.render(assigns) %>
```

Comment:
should this be `@inner_content` instead?

Reply:
You are right. `@inner_content` is the recommended way of rendering templates.